plugins/support-helphub/inc/helphub-customroles/helphub-custom-roles.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
```

Comment:
Let's clear out the plugin headers, they're part of a bigger plugin, so not needed.